video/out/vo_gpu_next.c

```diff
@@ -1515,84 +1521,216 @@ static void wait_events(struct vo *vo, int64_t until_time_ns)
     }
 }
 
-#if PL_API_VER < 342
-static inline void xor_hash(void *hash, pl_cache_obj obj)
+static char *cache_filepath(void *ta_ctx, char *dir, const char *prefix, uint64_t key)
 {
-    *((uint64_t *) hash) ^= obj.key;
+    bstr filename = {0};
+    bstr_xappend_asprintf(ta_ctx, &filename, "%s_%016" PRIx64, prefix, key);
```

Comment:
bikeshed: I think a dot would be nicer

Reply:
Indeed, but I just tried and it doesn't work well. 
`<hash>.icc` is associated as ICC profile on Windows and it is silly to name it that.
`icc.<hash>` is interpreting `<hash>` as file type, so again silly

Do you have good naming pattern in mind?

I added the prefix to make filenames more explicit and don't touch other files in cache directory.

Reply:
> Do you have good naming pattern in mind?

No, this is fine then.